### ISSUE TITLE
Preferences > Shortcuts: wrong focus in Shortcuts pref

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#1561]
**Describe the bug**
On Windows when launching preferences the focus is on the "View by" radio buttons and not the presences selector.
This particularly affects Non-mouse users, VI users.

This was originally logged on Bugzilla as P3 2691
https://bugzilla.audacityteam.org/show_bug.cgi?id=2691

**To Reproduce**
1. Open preferences
2. Select the Shortcuts category.
3. Press the OK button.
4. Reopen preferences.
5. Observe: one of the "View by" radio buttons is the focus. 

**Expected behavior**
The "Shortcuts" item in the categories tree should be the focus.

**Additional information (please complete the following information):**
 - OS: Windows 10
 - Version: Audacity 3.0.4 and earlier

**Additional context**
David Bailes wrote on Bugzilla:
>When the preferences dialog opens, the initial focus should always be an item in the categories tree. This is not the case - see steps to reproduce.
This bug was not present in Audacity 2.4.1, but is present by the time of the commit 1f366ee0730f752da.
It looks like it may have been caused by the switch to using wxWidgets 3.1.3 - no idea why.

And subsequently he wrote:
>This appears to be a wxWidgets bug.
Found this commit, which is included in wxWidgets 3.1.4, which is in the same area:
https://github.com/wxWidgets/wxWidgets/commit/68d7cb50822451b393f34680417d85c2354783cb
I've build Audacity with wxWidgets 3.1.4, and this fixes the problem.

**Comment by dozzzzer** [comment]
Not reproducible anymore: now, the first item in the tree - Audio Settings - is in focus when opening Preferences.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
